test-suite/data/credential--vc.transmute.world--did:web:vc.transmute.world#z6MksHh7qHWvybLg5QTPPdG2DgEjjduBDArV9EF9mRiRzMBN--CertifiedMillTestReport.json

```diff
@@ -0,0 +1,211 @@
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://w3c-ccg.github.io/vc-examples/cmtr/examples/v0.1/cmtr-v0.1.jsonld"
+  ],
+  "id": "http://localhost:3000/app/credential-did/did:elem:EiAsRb0lUsZbTgobLKOGV-CesWOcHrp6RaRiIH7Vxvmu0g",
+  "type": [
+    "VerifiableCredential",
+    "CertifiedMillTestReport"
+  ],
+  "issuer": "did:web:vc.transmute.world",
+  "issuanceDate": "2020-03-09T18:19:10.033Z",
+  "credentialSubject": {
+    "id": "did:elem:EiAsRb0lUsZbTgobLKOGV-CesWOcHrp6RaRiIH7Vxvmu0g",
+    "cmtr": {
+      "companyName": "Steel Inc.",
+      "companyWebsite": "https://transmute.industries",
+      "companyAddress": "3260 46 Ave SE #30, Calgary, AB T2B 3K7, Canada",
+      "companyPhoneNumber": "555 555 5555",
+      "companyBrandMark": "https://www.freelogodesign.org/file/app/client/thumb/e2b4bee1-4228-4cf5-b6d7-148c4ad93a9b_200x200.png?1583363125286",
```

Comment:
This comment should not hold up this PR. Hotlinking images may open up a privacy hole. It could be mitigated by using a trusted proxy to retrieve images on the client.

Reply:
This is a really excellent point.